types/rebass/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Rebass 0.3.8
+// Type definitions for Rebass 0.3.9
```

Comment:
Reviewing again what I did, I think this should be `// Type definitions for Rebass 3.0.1` no?

Reply:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `// Type definitions for rebass 3.0`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!